data/team.yaml

```diff
@@ -0,0 +1,14 @@
+our_team:
+  - idn: "001"
+    name: "Joe Thoresen"
+    position: "Chairman & Founder"
+    image: "/images/joe.jpg"
+    about: "Joe Thoresen, Cornerstone’s founding consultant, became a licensed industrial/ organizational psychologist in 1976. After graduate school, he joined the team at Rohm and Haas, where he specialized in hiring, selection, and management training. Joe attained the title of Manager of Training and Organization Development before leaving to begin a consulting career. He was one of the first consultants to specialize in applications of the assessment center method. In 1983, Joe founded Cornerstone Management Resource Systems, a company focused on applying real-world experience and published research to improvements in assessment center methodology. This effort grew into the creation of an internal promotion/development process and an entirely different type of assessment center."
+  - idn: "002"
```

Comment:
`idn` = id#, I needed a unique identifier to name the modal, so the proper modal lauches when clicked.